### PR TITLE
Add GTM routes

### PR DESCRIPTION
**Pull request by saschein**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json [hunk 1]
```diff
@@ -2206,8 +2336,8 @@
         "tags": [
           "Alerts"
         ],
-        "operationId": "Alerts_GetByEnrollment",
-        "description": "Gets the alert for an enrollment by alert ID.",
+        "operationId": "Alerts_GetByBillingAccount",
```
Comment (sarangan12) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json]: This is a breaking change. Could you please explain why this change was made in an existing version?

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json (hunk 1 above)
Comment (sarangan12) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json]: @saschein Please respond.

Comment (saschein) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json]: Sorry,
We are still in private-preview and are currently only using this api-version, we wanted to prevent adding a new one before we stabilize this version.
If this change must be in a new api-version we'll discard it and create a new PR later.

Thanks

Comment (sarangan12) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json]: Yes. Normally, we do not encourage breaking changes. It would be advisable to do the changes in new version

Comment (sarangan12) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json]: @saschein As mentioned in the previous comment, I am closing this PR. Please open a .new one with correct version
